view/templates/field_intcheckbox.tpl

```diff
@@ -1,8 +1,8 @@
 
-	
+
 	<div class='field checkbox'>
 		<label for='id_{{$field.0}}'>{{$field.1}}</label>
-		<input type="checkbox" name='{{$field.0}}' id='id_{{$field.0}}' value="{{$field.3}}" {{if $field.2}}checked="true"{{/if}} aria-describedby='{{$field.0}}_tip'>
+		<input type="checkbox" name='{{$field.0}}' id='id_{{$field.0}}' value="{{$field.3}}" {{if $field.2}}checked{{/if}} aria-describedby='{{$field.0}}_tip'>
```

Comment:
Please fix the quotation marks in the HTML attributes while you’re at it.
```suggestion
		<input type="checkbox" name="{{$field.0}}" id="id_{{$field.0}}" value="{{$field.3}}" {{if $field.2}}checked{{/if}} aria-describedby="{{$field.0}}_tip">
```

Reply:
Sure, done.